www/js/messages.js

```diff
@@ -19,7 +17,7 @@ function initMessages(head, query) {
   var loadingMessages = false;
 
   // Set group name
-  head.find('#group-name').html(query.groupName);
+  head[0].f7RelatedNavbar.children['group-name'].innerHTML = query.groupName;
```

Comment:
Almost. Another bug appeared with this line so I gave up for the time being. Gonna reset it in this pr, just like my other changes to messages